dist/docker-images/ziti-controller/compose.test.bash

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
```

Comment:
this worked great once i stopped my local controller. might be worthwhile to test for port 1280 being open first before continuing, but overall this worked really well. thanks for changing it.

Reply:
Good point. There's no reason for this test to require all interfaces since it's only using 127.21.71.0, and it can use a non-standard port to further reduce the chance of a collision. Checking for an open port is also doable, but makes the script less portable and slightly more complicated. I'll take a stab at a minimal port check.